frontend/src/views/DashboardManager/NoPlasticCard.vue

```diff
@@ -7,9 +7,9 @@
       <h3 class="fr-text font-weight-bold">{{ keyMeasure.shortTitle }}</h3>
     </v-card-title>
     <v-card-text v-if="needsData">
-      <DataInfoBadge :missingData="needsData" class="py-0 ml-8" />
+      <DataInfoBadge :missingData="true" class="py-0 ml-8" />
```

Comment:
I'm guessing when we decide to implement the badges we would change this hard-coded `true` right?

Reply:
The hardcoded true exists just because at the level above there is a v-if. There are a couple of different flags for the component so I'm not sure what refactoring we would do?